packages/plugins/users-permissions/admin/src/pages/Roles/EditPage/index.js

```diff
@@ -70,16 +84,17 @@ const EditPage = () => {
         onSubmit={handleCreateRoleSubmit}
         validationSchema={schema}
       >
-        {({ handleSubmit }) => (
-          <form onSubmit={handleSubmit}>
+        {({ handleSubmit, values, handleBlur, handleChange, errors }) => (
+          <>
```

Comment:
Your content should be wrapped in the `Form` component instead of a Fragment.

```
import { Form } from '@strapi/helper-plugin';

{({ errors,  handleChange, values }) => (
  <Form noValidate>
     <HeaderLayout />
     {....}
  </Form>
)})}
```